cli/src/main/kotlin/StartCommand.kt

```diff
@@ -45,41 +48,33 @@ class StartCommand(private val config: OrtServerOptions) : SuspendingCliktComman
         help = "The ID of the repository."
     ).long().required()
 
-    private val revision by option(
-        "--vcs-revision",
-        envvar = "VCS_REVISION",
-        help = "Revision of the repository to start the run for."
-    ).required()
-
-    private val path by option(
-        "--vcs-sub-path",
-        envvar = "VCS_SUB_PATH",
-        help = "Optional VCS sub-path of the repository."
-    )
-
-    private val jobConfigContext by option(
-        "--job-config-context",
-        envvar = "JOB_CONFIG_CONTEXT",
-        help = "Configuration context to use for the run."
-    )
-
     private val wait by option(
         "--wait",
         envvar = "WAIT",
         help = "Wait for the run to finish."
     ).flag()
 
+    private val parameters by mutuallyExclusiveOptions(
+        option(
+            "--parameters-file",
+            envvar = "PARAMETERS_FILE",
```

Comment:
I think that `PARAMETERS_FILE` is too generic for an environment variable.
At the time the user configures these env vars, they are not in the `run start` context.

For the download commands I solved this issue by using a path with an ORT prefix:
`ORT_DOWNLOAD_LOGS_OUTPUT_DIR`

I know that this isn't completely consistent everywhere (e.g. we still use `REPOSITORY_ID` instead of `ORT_REPOSITORY_ID`).

What do you think about `ORT_RUNS_START_PARAMETERS_FILE`?